src/skill_tree.md

```diff
@@ -179,10 +184,9 @@ href = "https://github.com/rust-lang/rust/issues/74925"
 items = []
 
 [[group]]
-name = "rfc"
```

Comment:
this used to contain more RFCs, but they have been moved to tracking issues since. I was thinking that we should somehow highlight that some node is an RFC and not something already accepted (though... considering the random collection of issues I opened that doesn't have a merged RFC, I guess we can just treat them the same)